src/backend/partaj/core/migrations/0048_increase_objectfield_referral.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.5 on 2021-11-18 08:16
+
+from django.db import migrations, models
+import django_fsm
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0047_fix_existing_published_answers'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='referral',
+            name='object',
+            field=models.CharField(blank=True, help_text='Brief sentence describing the object of the referral', max_length=60, verbose_name='object'),
+        ),
+        migrations.AlterField(
```

Comment:
I don't think you meant to have a migration related to referral state in this PR